opensearch/src/test/java/org/opensearch/sql/opensearch/data/value/OpenSearchExprGeoPointValueTest.java

```diff
@@ -17,29 +15,8 @@ class OpenSearchExprGeoPointValueTest {
 
   private OpenSearchExprGeoPointValue geoPointValue = new OpenSearchExprGeoPointValue(1.0, 1.0);
 
-  @Test
-  void value() {
-    assertEquals(new OpenSearchExprGeoPointValue.GeoPoint(1.0, 1.0), geoPointValue.value());
-  }
-
   @Test
   void type() {
     assertEquals(OpenSearchGeoPointType.of(), geoPointValue.type());
   }
```

Comment:
Add a test to get value
And a test to prove compatibility with STRUCT

Reply:
Added test to get value. Did not add test to prove compatibility with STRUCT because it is not compatible with struct.